### DISCUSSION TITLE
Where freac store configuration files?

### DISCUSSION THREAD

**Discussion by rmlazzari** [enzo1982/freac#648] [Help & Support] [answered]
Hi, I'd like to install a new HD in my PC and I wish to bring freac conf files to the new device. I have freac v.1.1.6 not installed but as an appimage (freac-1.1.6-linux-x86_64.AppImage) under Debian 12.

Could you help with that? Thank!

**Comment by cleanyoursockets** [comment, marked as answer]
Hello, I also use an Appimage of Freac with Debin 12. When in your home folder, type ctrl + H, there you can find a Freac folder under .config
Copy that file to your new installation, good luck!

**Reply by rmlazzari** [reply]
Perfect! Thank you, cleanyoursockets!

**Comment by Bammede** [comment]
I installed it thru homebrew on mac, but can not find config file

**Reply by enzo1982** [reply]
On macOS, config files can usually be found in ~/Library/Preferences/freac/
